source/winVersion.py

```diff
@@ -28,36 +30,43 @@ def isUwpOcrAvailable():
 
 
 WIN10_VERSIONS_TO_BUILDS = {
-	1507: 10240,
-	1511: 10586,
-	1607: 14393,
-	1703: 15063,
-	1709: 16299,
-	1803: 17134,
-	1809: 17763,
-	1903: 18362,
-	1909: 18363,
-	2004: 19041,
-	2009: 19042,
+	"15H1": 10240,
+	"15H2": 10586,
+	"16H2": 14393,

```

Comment:
@DHowett or @miniksa, are these correct (or at least sane) identifiers for the 2015 and 2016 versions of Windows 10?

As an aside, how do you comment on a range of lines using the keyboard?

Reply:
Nope - see an earlier comment.